### PR TITLE
Code Refactoring and composer configuration

### PR DESCRIPTION
**Pull request by chetan-thapliyal**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +48,8 @@ before_script:
   - composer require --dev typo3/cms="$TYPO3_VERSION"
   # Restore composer.json
   - git checkout composer.json
+  - export UNIT_BOOTSTRAP='.Build/vendor/nimut/testing-framework/res/Configuration/UnitTestsBootstrap.php'
```
Comment (tomasnorre) [inline .travis.yml]: Doesn't the Configuration/UnitTests.xml not include the bootstrapping ?

File: Classes/Service/Eid.php [hunk 2]
```diff
@@ -24,72 +24,10 @@
  *  This copyright notice MUST APPEAR in all copies of the script!
  ***************************************************************/
 
-use TYPO3\CMS\Core\Utility\GeneralUtility;
+\TYPO3\CMS\Frontend\Utility\EidUtility::initTCA();
 
-/**
```
Comment (tomasnorre) [inline Classes/Service/Eid.php]: Looks like the class-definition have been remove..

File: composer.json [hunk 3]
```diff
@@ -31,18 +31,44 @@
...
+            "[ -e 'vendor/bin/phpunit' ] || composer cleanupdate",
+            "vendor/bin/phpunit --colors -v -c vendor/nimut/testing-framework/res/Configuration/UnitTests.xml Tests/Unit"
+        ],
+        "test:functional": [
+            "[ -e 'vendor/bin/phpunit' ] || composer cleanupdate",
+            "mkdir -p typo3conf/ext",
+            "ln -snvf ../../. typo3conf/ext/feature_flag",
+            "vendor/bin/phpunit --colors -v -c vendor/nimut/testing-framework/res/Configuration/FunctionalTests.xml Tests/Functional"
+        ],
+        "test:params": [
```
Comment (tomasnorre) [inline composer.json]: Please remove the test:params block was only used for PoC in the composer meeting.

File: .travis.yml (hunk 1 above)
Comment (chetan-thapliyal) [reply .travis.yml]: This configuration was required previously. After refactoring the tests this is no longer required. Using "Configuration/UnitTests.xml" is enough now.

File: Classes/Service/Eid.php (hunk 2 above)
Comment (chetan-thapliyal) [reply Classes/Service/Eid.php]: The class has now been moved to its own file. The eID script and processor class now reside in individual files.

**Pull request merged**